hugr-py/src/hugr/build/dfg.py

```diff
@@ -195,12 +195,19 @@ def add_op(
 
         return replace(new_n, _num_out_ports=op.num_out)
 
-    def add(self, com: ops.Command, *, metadata: dict[str, Any] | None = None) -> Node:
+    def add(
+        self,
+        com: ops.Command,
+        *,
+        static_in: Iterable[Wire] | None = None,
+        metadata: dict[str, Any] | None = None,
+    ) -> Node:
```

Comment:
Add the `static_in` argument to `add_op` too (and maybe to `_wire_up`), and use it here.